db/tasks.php

```diff
@@ -71,5 +71,14 @@
         'dayofweek' => '*',
         'month'     => '*'
     ),
+    array(
+        'classname' => 'tool_objectfs\task\check_objects_location',
+        'blocking'  => 0,
+        'minute'    => '7',
```

Comment:
7 seems a little arbitrary, what is the purpose of this? If it is to spread load and not hit the 0th minute then a value of "R" for random is probably better

https://docs.moodle.org/dev/Task_API#Cron_syntax_examples